nixos/services/haproxy/frontend.nix

```diff
@@ -0,0 +1,132 @@
+# This role allows access to Kubernetes Service IPs (also called ClusterIPs, 10.0.0.0/24)
```

Comment:
this file shouldn't be here, belongs to the kubernetes role. Just remove it?

Reply:
Whoops, VSCode ist interressant